com.stansassets.build/Editor/Steps/IBuildStep.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace StansAssets.Build.Editor
+{
+    public interface IBuildStep
+    {
+        bool Execute(BuildContext buildContext);
+
+        int Priority { get; }
+        string GetResultMessage();
```

Comment:
it's not clear to me when the system will call this function for my `BuildStep`.
If you want users to provide the result we can add callback with the result into the `Execute ` this way it's clear for me as for API usaer how and when to provide the result. 
@alexey-yaremenko  would be nice to have your input here as well.